deps/rabbitmq_stomp/src/rabbit_stomp_processor.erl

```diff
@@ -7,85 +7,101 @@
 
 -module(rabbit_stomp_processor).
 
+-feature(maybe_expr, enable).
+
 -compile({no_auto_import, [error/3]}).
 
 -export([initial_state/2, process_frame/2, flush_and_die/1]).
 -export([flush_pending_receipts/3,
-         handle_exit/3,
          cancel_consumer/2,
-         send_delivery/5]).
+         handle_down/2,
+         handle_queue_event/2]).
 
 -export([adapter_name/1]).
 -export([info/2]).
 
--include_lib("amqp_client/include/amqp_client.hrl").
--include_lib("amqp_client/include/rabbit_routing_prefixes.hrl").
 -include("rabbit_stomp_frame.hrl").
 -include("rabbit_stomp.hrl").
 -include("rabbit_stomp_headers.hrl").
+-include_lib("rabbit/include/amqqueue.hrl").
+-include_lib("amqp_client/include/amqp_client.hrl").
 
--record(proc_state, {session_id, channel, connection, subscriptions,
-                version, start_heartbeat_fun, pending_receipts,
-                config, route_state, reply_queues, frame_transformer,
-                adapter_info, send_fun, ssl_login_name, peer_addr,
-                %% see rabbitmq/rabbitmq-stomp#39
-                trailing_lf, auth_mechanism, auth_login,
-                default_topic_exchange, default_nack_requeue}).
-
--record(subscription, {dest_hdr, ack_mode, multi_ack, description}).
+-record(proc_state, {session_id, subscriptions,
+                     version, start_heartbeat_fun, pending_receipts,
```

Comment:
My editor shows that `start_heartbeat_fun` is an unused field. Where is this field used? Remove any unused fields.